### ISSUE TITLE
Cannot get the lat long of an region

### ISSUE THREAD

**Issue by AbhaysinghBhosale** [jsonmaur/aws-regions#11]

**Comment by Sumo99** [comment]
Here are the regions with lat and long:

us-east-1;Virginia;38.13;-78.45
us-east-2;Ohio;39.96;-83
us-west-1;California;37.35;-121.96
us-west-2;Oregon;46.15;-123.88
eu-west-1;Ireland;53;-8
eu-west-2;London;51;-0.1
eu-west-3;Paris;48.86;2.35
eu-central-1;Frankfurt;50;8
sa-east-1;Sao Paulo;-23.34;-46.38
ap-southeast-1;Singapore;1.37;103.8
ap-southeast-2;Sydney;-33.86;151.2
ap-northeast-1;Tokyo;35.41;139.42
ap-northeast-2;Seoul;37.56;126.98
ap-south-1;Mumbai;19.08;72.88
ca-central-1;Canada Central;45.5;-73.6

**Comment by AbhaysinghBhosale** [comment]
Is there any api to get the lat long of regions

**Comment by Sumo99** [comment]
I added a pull request. I simply added this stuff to the json file.

**Comment by AbhaysinghBhosale** [comment]
Are this the real exact locations and what about new coming regions

**Comment by jsonmaur** [comment]
Hey guys, thanks for filing the issue and the PR. I'm not sure what the purpose of adding these coordinates would be. I'm assuming they are not the exact coordinates of the data centers, but rather the general coordinates of the country/state?

If you could explain how you plan on using them, maybe we can tweak the data structure to be a little clearer as to what the coordinates represent (and possibly add functions to the SDKs to find the closest region to a location, for example.)

**Comment by AbhaysinghBhosale** [comment]
Hi there,
I am working on the platform where I need to show the map of customers aws
resources locations in map.For this purpose I am looking for an api which
will give the lat long of data center for the resource.

This ask provides locations but it's difficult to trust it as this may not
be exact coordinates and it's after all headcoded.

**Comment by Sumo99** [comment]
Hi jsonmaur,

I think these lat and longs are the locations of the datacenters themselves, see https://stackoverflow.com/a/8668187. Maybe it will be helpful for latency calculations?

**Comment by AbhaysinghBhosale** [comment]
You are may be the correct but this not provided by aws itself.

**Comment by jsonmaur** [comment]
Yeah, I think latency calculations are much more complicated than simply finding the nearest data center. Might be useful to add in the future. I do see @AbhaysinghBhosale's point in the coordinates being useful for showing regions on a map.

**Comment by Seungwoo321** [comment]
I think it is important to have a coordinate that can show the approximate location.
